script.js

```diff
@@ -27,6 +27,51 @@ crossmenu.addEventListener('click', () => {
   multipost.classList.add('Menu-mobile-after');
 });
 
+// Form Validation L
+const form = document.getElementById('form');
+const emailForm = 'ERROR enter a valid email address';
```

Comment:
- [ ] [_Optional_] According to your project requirement, you're to display an error message informing the user of their error/mistake but here, you displayed "ERROR enter a valid email address". The email is actually valid, it just has to contain lower case characters. Kindly change the error message.
- [ ] [_Optional_] According to your project requirement, you're to display the error message near the submit button but yours is not near the submit button